notebooks/00_quick_start/fastai_recommendation.ipynb

```diff
@@ -27,12 +27,12 @@
      "name": "stdout",
```

Comment:
do we need to run all these notebooks again? In the fastai one, we are not using GPU